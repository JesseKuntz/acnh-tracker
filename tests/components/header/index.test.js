import { h } from 'preact';
import Header from '../../../src/components/header';

import { shallow } from 'enzyme';

describe('Header', () => {
  test('matches snapshot', () => {
    expect(shallow(<Header />)).toMatchSnapshot();
  });
});
