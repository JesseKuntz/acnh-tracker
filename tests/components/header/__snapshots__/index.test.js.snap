// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches snapshot 1`] = `
<div
  className="header-container"
>
  <header
    className="header-primary header"
  >
    <div
      className="content"
    >
      <Link
        activeClassName="active"
        href="/"
        onClick={[Function]}
      >
        <h1>
          AC Tracker
        </h1>
        <img
          className="check"
          src="../../assets/menu/check.svg"
        />
      </Link>
      <div
        className="hamburger-buns"
      >
        <div
          className={null}
          onClick={[Function]}
        >
          <div
            className="bar1"
          />
          <div
            className="bar2"
          />
          <div
            className="bar3"
          />
        </div>
      </div>
    </div>
  </header>
  <header
    className="header-menu header"
  >
    <nav
      className="hide"
    >
      <HeaderLink
        clickHandler={[Function]}
        type="fish"
      />
      <HeaderLink
        clickHandler={[Function]}
        type="bug"
      />
      <HeaderLink
        clickHandler={[Function]}
        type="sea-creature"
      />
      <HeaderLink
        clickHandler={[Function]}
        href="/monthly"
        type="monthly-catches"
      />
    </nav>
  </header>
  <header
    className="header-secondary header"
  >
    <div
      className="content"
    >
      <LoginInfo />
    </div>
  </header>
</div>
`;
